### PR TITLE
Update Readme

### LINKED ISSUE #260: Update repo ReadMe.md file
In line with the issue available [here](https://github.com/BHoM/admin/issues/4), please update your repositories ReadMe.md file to match the template provided in the linked admin issue.

Please remember to change any instances of `Robot_Toolkit` in the template to your repository name.

I will comment separately with a specific AppVeyor badge link that should be used for this repo.

When you resolve this issue, please add myself and @al-fisher as reviewers.

### PR DESCRIPTION
**Pull request by adecler**
# Issues addressed by this PR

Closes #260

Updating the Readme following the changes already made on BHoM adapter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,31 @@
 [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![Build status](https://ci.appveyor.com/api/projects/status/f4lrwf0x1uclsort/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/bhom_ui/branch/master)
+[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/BHoM_UI/BHoM_UI.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=17&branchName=master)
```
Comment (adecler) [inline README.md:3]: What's the definitionId for BHoM_UI ?

Comment (FraserGreenroyd) [reply README.md:3]: `19` - full badge is: `[Build Status](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=19&branchName=master)`

Comment (adecler) [reply README.md:3]: To avoid asking the same question for each PR, how can I find the number for each repo ?

**Comment by adecler** [comment]
I have fixed the definitionId so it should be ready for review

**Pull request merged**